pallets/parachain-system/Cargo.toml

```diff
@@ -34,6 +34,7 @@ xcm = { git = "https://github.com/paritytech/polkadot", default-features = false
 cumulus-pallet-parachain-system-proc-macro = { path = "proc-macro", default-features = false }
 cumulus-primitives-core = { path = "../../primitives/core", default-features = false }
 cumulus-primitives-parachain-inherent = { path = "../../primitives/parachain-inherent", default-features = false }
+cumulus-pallet-aura-ext = { path = "../aura-ext", default-features = false }
```

Comment:
Should it be feature-gated?

Reply:
Dependency is backwards. Aura-ext should depend on parachain-system and implement `ConsensusHook` in its own code.

This is a general principle in Rust development: expose traits in one crate and implement them in others.